streamer/src/quic.rs

```diff
@@ -514,7 +516,14 @@ mod test {
             .with_safe_defaults()
             .with_custom_certificate_verifier(SkipServerVerification::new())
             .with_no_client_auth();
-        ClientConfig::new(Arc::new(crypto))
+        let mut config = ClientConfig::new(Arc::new(crypto));
+
+        let transport_config = Arc::get_mut(&mut config.transport).unwrap();
+        let timeout = IdleTimeout::from(VarInt::from_u32(60_000));
+        transport_config.max_idle_timeout(Some(timeout));
```

Comment:
Do we need a corresponding change on the client side as well? As per `max_idle_timeout()` documentation:
```
The true idle timeout is the minimum of this and the peer's own max idle timeout.
```

Reply:
yea.